server/db/1_setup_posts.sql

```diff
@@ -0,0 +1,18 @@
+DO
+$$
+BEGIN
+    IF NOT EXISTS(SELECT *
+                   FROM information_schema.tables
+                   WHERE table_schema = 'posts'
+                         AND table_name = 'posts')
+    THEN
+        CREATE TABLE posts (
+            id serial PRIMARY KEY,
+            title varchar(50) NOT NULL,
+            name varchar(10) NOT NULL,
+            body varchar NOT NULL,
+            user_id int
+        );
+  END IF;
+END;
+$$
```

Comment:
Not sure whats going on here, or what couldn't be achieved by just dropping the table first then making it.

Reply:
this helps with keeping the existing data we can display all the posts